dashboard/test/models/pd/workshop_test.rb

```diff
@@ -1022,14 +1022,19 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     assert_nil @workshop.processed_location
   end
 
-  test 'suppress_reminders?' do
+  test 'suppress_reminders? is true for certain subjects by default' do
     suppressed = [
       create(:fit_workshop, course: Pd::Workshop::COURSE_CSF),
       create(:workshop, course: Pd::Workshop::COURSE_CSD, subject: Pd::Workshop::SUBJECT_CSD_TEACHER_CON),
       create(:fit_workshop, course: Pd::Workshop::COURSE_CSD),
       create(:workshop, course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_TEACHER_CON),
       create(:fit_workshop, course: Pd::Workshop::COURSE_CSP),
-      create(:admin_counselor_workshop, course: Pd::Workshop::COURSE_ADMIN_COUNSELOR),
+      create(:admin_counselor_workshop, subject: Pd::Workshop::SUBJECT_ADMIN_COUNSELOR_WELCOME),
+      create(:admin_counselor_workshop, subject: Pd::Workshop::SUBJECT_ADMIN_COUNSELOR_SLP_INTRO),
+      create(:admin_counselor_workshop, subject: Pd::Workshop::SUBJECT_ADMIN_COUNSELOR_SLP_CALL1),
+      create(:admin_counselor_workshop, subject: Pd::Workshop::SUBJECT_ADMIN_COUNSELOR_SLP_CALL2),
+      create(:admin_counselor_workshop, subject: Pd::Workshop::SUBJECT_ADMIN_COUNSELOR_SLP_CALL3),
+      create(:admin_counselor_workshop, subject: Pd::Workshop::SUBJECT_ADMIN_COUNSELOR_SLP_CALL4)
```

Comment:
in order for this test case to be testing what you say it is, I think you would need to stop setting `suppress_email` here: https://github.com/code-dot-org/code-dot-org/blob/e45e588889ff46e4b20f57307331cb69129a14de/dashboard/test/factories/workshop_factories.rb#L292

Reply:
related: since we are turning on reminders for AYWs, it seems like we should remove this line too? https://github.com/code-dot-org/code-dot-org/blob/e45e588889ff46e4b20f57307331cb69129a14de/dashboard/test/factories/workshop_factories.rb#L163

Reply:
Ah yes, strong point––will update